docs/fundamentals/code-analysis/quality-rules/singlefile-warnings.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Publish rules (code analysis)
```

Comment:
For file renames, we also need to add a redirect to the redirection JSON file in the root of the repo.

Reply:
I modified the link in the categories.md and toc.yml. I didn't found other occurrence of "publish-warnings" in the repo

Reply:
It's also in case people have bookmarked that page or it's linked from elsewhere.